Src/HALAL/Services/Communication/Ethernet/EthernetNode.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * EthernetNode.cpp
+ *
+ *  Created on: Nov 23, 2022
+ *      Author: stefa
+ */
+
+#include "Communication/Ethernet/EthernetNode.hpp"
+
+EthernetNode::EthernetNode(IPV4 ip, uint32_t port):ip(ip), port(port){}
+
+bool EthernetNode::operator==(const EthernetNode& other) const{
+	return ip.ip_address.addr == other.ip.ip_address.addr && port == other.port;
```

Comment:
```suggestion
	return ip.address.addr == other.ip.address.addr && port == other.port;
```
Con lo que te he marcado arriba el naming quedaria así, mas limpio